### ISSUE TITLE
Diagram explaining forward/back keyboard navigation in training should be removed.

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#503]
In user tests so far, no one has gotten the intended meaning out of that diagram, and even when it is discussed explicitly, no one has yet actually opted to use that mode of navigation.

We should simply remove the guider diagram, and leave keyboard navigation as a feature that users can discover on their own.
